.github/workflows/main.yml

```diff
@@ -0,0 +1,25 @@
+---
+name: CI
+
+on:
+  pull_request:
+  push:
+    branches:
+      - 'master'
+      - '*-stable'
+
+concurrency:
+  group: ${{ github.ref_name }}-${{ github.workflow }}
+  cancel-in-progress: true
+
+jobs:
+  rubocop:
+    name: Rubocop
+    uses: theforeman/actions/.github/workflows/rubocop.yml@v0
+
+  test:
+    name: Ruby
+    needs: rubocop
+    uses: theforeman/actions/.github/workflows/smart_proxy_plugin.yml@proxy_plugin
+    with:
+      foreman_proxy_version: enter-the-matrix
```

Comment:
I think we need to make the `Gemfile` respect this and the URL because now it's hardcoded:
https://github.com/theforeman/smart_proxy_pulp/blob/a5e48e24c69dd557c279f7a5c1896d35edb32d9a/Gemfile#L17

Reply:
implemented